packages/iTwinUI-react/src/core/NotificationMarker/NotificationMarker.tsx

```diff
@@ -0,0 +1,78 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { CommonProps, useTheme } from '../utils';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/notification-marker.css';
+
+export type NotificationMarkerProps = {
+  /**
+   * Content of the NotificationMarker.
+   */
+  children: React.ReactNode;
+  /**
+   * Type of notification
+   *
+   * 'primary' = blue,
+   * 'positive' = green,
+   * 'warning' = orange,
+   * 'negative' = red,
+   * @default 'primary'
+   */
+  type?: 'primary' | 'positive' | 'warning' | 'negative';
```

Comment:
I think this prop should be called `status`

Reply:
**[DONE]** Ohh, I think I got confused with the `type` from `Alert.tsx`. Changed it to `status` now

![image](https://user-images.githubusercontent.com/45748283/191323870-7dee5a32-0777-4061-86d6-f6115a3ff8ed.png)